### ISSUE TITLE
您这样写是不是有点问题

### ISSUE THREAD

**Issue by EnnengYang** [ShinKyuY/Graph-Neural-Net#3]
def loss(self, lr, active=tf.nn.relu,optim=tf.train.AdamOptimizer):
        self.L=tf.identity(self.X)
        for i in range(len(self.layer)):
            self.L=self.graph_conv(self.L, self.adj, self.layer[i], activation=active)
            self.L= tf.nn.dropout(self.L, self.dropout)
            
        self.hypothesis=tf.layers.dense(tf.matmul(self.adj,self.L), self.output_dim, activation=None)

你每次调用loss的时候都会创建新的tf.layers.dense    那么每次网络的权重都是新的
